feat(documentation): add stories for the list utilities (#1060)
swisspost/design-system: packages/documentation/src/stories/utilities/list/list.stories.mdx

  }}
/>

# Lists

<p className="lead">HTML lists allow web developers to group a set of related items in lists.</p>

There are three main types of lists, from which you can choose the one that best suits your needs.
Our utilities give you the possibility to visualize them in different ways.

## Unordered List

The HTML `<ul>` tag defines an unordered (bulleted) list.

<Canvas>
  <Story name="Unordered List">
    <ul>
      <li>This is an unorderd list.</li>
      <li>It appears in its default style.</li>
      <li>

>         Therefore it should be rendered with a bulletpoint at the beginning of each list item.

Reviewer: ```suggestion
        Therefore it is rendered with a bullet point in front of each list item.
```
